strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/RepositoryVersionValidatorsManagementControllerTest.java

```diff
@@ -28,153 +31,137 @@
-                          .statusCode(200);
-
-        RestAssuredMockMvc.given()
-                          .header("Accept", "application/json")
-                          .when()
-                          .get("/configuration/repositories/version-validators/storage0/all-validators")
-                          .peek()
-                          .then()
-                          .body(equalTo("[ \"REDEPLOYMENT\", \"RELEASE\" ]"))
-                          .statusCode(200);
-
-        RestAssuredMockMvc.given()
-                          .header("Accept", "application/json")
-                          .when()
-                          .delete("/configuration/repositories/version-validators/storage0/all-validators/SNAPSHOT")
+                          .delete("/configuration/repositories/version-validators/storage0/releases-with-default-validators/maven-snapshot-version-validator")
                           .peek()
                           .then()
                           .statusCode(200);
 
         RestAssuredMockMvc.given()
                           .header("Accept", "application/json")
                           .when()
-                          .get("/configuration/repositories/version-validators/storage0/all-validators")
+                          .get("/configuration/repositories/version-validators/storage0/releases-with-default-validators")
                           .peek()
                           .then()
-                          .body(equalTo("[ \"REDEPLOYMENT\", \"RELEASE\" ]"))
+                          .body(anyOf(equalTo("[ \"redeployment-validator\", \"maven-release-version-validator\" ]"),
+                                      equalTo("[ \"maven-release-version-validator\", \"redeployment-validator\" ]")))
                           .statusCode(200);
     }
 
     @Test
     public void validatorsForReleaseRepositoryShouldBeAdditableAndFailSafe()
-            throws Exception
     {
         RestAssuredMockMvc.given()
                           .header("Accept", "application/json")
                           .when()
-                          .get("/configuration/repositories/version-validators/storage0/single-validator-only")
+                          .get("/configuration/repositories/version-validators/storage0/releases-with-single-validator")
                           .peek()
                           .then()
-                          .body(equalTo("[ \"REDEPLOYMENT\" ]"))
+                          .body(equalTo("[ \"redeployment-validator\" ]"))
                           .statusCode(200);
 
         RestAssuredMockMvc.given()
                           .header("Accept", "application/json")
                           .when()
-                          .put("/configuration/repositories/version-validators/storage0/single-validator-only/SNAPSHOT")
+                          .put("/configuration/repositories/version-validators/storage0/releases-with-single-validator/maven-snapshot-version-validator")
```

Comment:
@fuss86 ,

In the future, could I ask you to please use more "repository-like" names for the repositories? The first few times I saw these repositories, I thought that some tests were messing up some paths and they I ended up digging into the code to see what was going on! :D